config/brokers/mt-channel-broker/deployments/broker-filter.yaml

```diff
@@ -92,6 +95,11 @@ spec:
             value: "8080"
         securityContext:
           allowPrivilegeEscalation: false
+          readOnlyRootFilesystem: true
+          runAsNonRoot: true
```

Comment:
some of this was already done before and got reverted - see: https://github.com/knative/eventing/pull/5629/files

Reply:
FWIW we reverted because we didn't have time - at the time - to figure out mem/cpu requests that were low enough to work reliably in GitHub actions, the securityContext stuff should IMO be good either way, and I think having resources present - even if low - is better than not having them at all (so being entirely at the mercy of the scheduler happening not to collocate too many jobs on a node).

tl;dr imo we should just make sure this reliably works in GitHub actions, and lower mem/cpu until it does :)